frontend/src/utils/AuthUtils.ts

```diff
@@ -0,0 +1,27 @@
+import IdTokenVerifier from "idtoken-verifier";
+
+import { DecodedJWT } from "../types/AuthTypes";
+
+// No need to fill in the issuer and audience fields since we're not validating
+// the token, just decoding it.
+const idTokenVerifier = new IdTokenVerifier({
+  issuer: "",
+  audience: "",
```

Comment:
To be clear, the argument type unfortunately requires I pass SOMETHING so the empty string will have to do.